Maquetado/scss/styles.scss

```diff
@@ -1,3 +1,7 @@
 * {
   font-family: 'Roboto', sans-serif;
 }
+
+.body-container {
+  padding: $nav-height+50px 100px;
```

Comment:
I think it's better if we only apply the nav-height in the upper part and not in the bottom.
```
padding: $nav-height+50px 100px 50px;